openslides/users/static/templates/users/group-detail.html

```diff
@@ -0,0 +1,14 @@
+<h1>{{ group.name }}</h1>
+
+<div id="submenu">
+  <a ui-sref="users.group.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview
```

Comment:
Yes, I also think about if custom (closing) tags can be omit. I tested it with the <translation> tag. The translation will be extracted correctly. So for translation we could use the optional form. But if you really like the formal, valid variant I have to add the long form with </translation>.
What do you think?